glyphs-reader/src/font.rs

```diff
@@ -80,6 +80,8 @@ impl FeatureSnippet {
 pub struct Glyph {
     pub glyphname: String,
     pub layers: Vec<Layer>,
+    pub left_kern: Option<String>,
+    pub right_kern: Option<String>,
```

Comment:
I would probably either call these `left/right_kern_group` or else I would add a doc comment that these strings are the name of the group the glyph belongs to, if any?

Reply:
Broadly I'm trying to match modern glyphs naming so I will add a comment

Reply:
in that case it may be worth sticking with the original "kern_left" / "kern_right" even if it the reverse may look more correct in English